src/plugins/wb-postback/wb-postback.js

```diff
@@ -27,7 +27,7 @@ var $document = wb.doc,
 					wb.getData( $elm, componentName )
 				),
 				attrEngaged = "data-wb-engaged",
-				$buttons = $( "[type=submit]", $elm ),
+				$buttons = $( "[type=submit], button:not([type=button]):not([type=reset])", $elm ),
```

Comment:
Here simplified version

```suggestion
				$buttons = $( "[type=submit], button:not([type]])", $elm ),
```

This match the HTML5 specification: https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element:missing-value-default

Reply:
Good idea :)! I like this more than what I started out with since it's simpler and more future-proof.